### PR TITLE
Remove pre 3.3.x sections, just document current values

### PR DESCRIPTION
**Pull request by gschueler**

### CODE REVIEW

File: docs/administration/configuration/system-properties.md [hunk 1]
```diff
@@ -37,8 +35,8 @@ java -server -Dserver.servlet.session.timeout=3600 -Dserver.port=8080 -jar runde
...
 - `rundeck.jetty.connector.ssl.excludedCipherSuites` Comma-separated list of Cipher suites to disable. No default. See [Disabling SSL Protocols](/administration/security/ssl.md#disabling-ssl-protocols).
 - `rundeck.jetty.connector.ssl.includedCipherSuites` Comma-separated list of Cipher suites to enable. Default is based on available cipher suites. See [Disabling SSL Protocols](/administration/security/ssl.md#disabling-ssl-protocols).
-- `logging.config` The absolute path to your log4j2 configuraiton file. This is the Spring Boot log file setting and must be set in rpm,deb, and docker environments for the logging system to initialize properly (Rundeck 3.3.x)
-- `log4j.configurationFile` The absolute path to your log4j2 configuration file. (Rundeck 3.3.x)
+- `logging.config` The absolute path to your log4j2 configuration file. This is the Spring Boot log file setting and must be set in rpm,deb, and docker environments for the logging system to initialize properly
```
Comment (gschueler) [inline docs/administration/configuration/system-properties.md:38]: @sjrd218 can you clarify what this section says? are both config values necessary to set the logfile path? `logging.conf` vs `log4j.configurationFile` ?

File: docs/administration/configuration/system-properties.md [hunk 2]
```diff
@@ -18,12 +18,10 @@ java -server -Dserver.servlet.session.timeout=3600 -Dserver.port=8080 -jar runde
 - `server.port` The HTTP port to use for the server, default "4440"
 - `server.https.port` The HTTPS port to use or the server, default "4443"
 - `server.address` Address/hostname to listen on, default is "localhost"
-- `server.contextPath` Web context path to use, such as "/rundeck". Default is "/". (Pre-Rundeck 3.3.x)
```
Comment (gschueler) [inline docs/administration/configuration/system-properties.md:21]: I think any difference between 3.3.x and pre 3.3.x should be highlighted in the upgrade notes, not buried here @fdevans

Comment (fdevans) [reply docs/administration/configuration/system-properties.md:21]: agreed

File: docs/administration/configuration/system-properties.md (hunk 1 above)
Comment (sjrd218) [reply docs/administration/configuration/system-properties.md:38]: We should be good to remove reference to the log4j.configurationFile. The Spring boot config is the one we would want to document.

**Pull request merged**